go.mod

```diff
@@ -8,11 +8,12 @@ require (
 	github.com/onsi/gomega v1.16.0
 	github.com/openshift/hive/apis v0.0.0-20210910175845-0756b70e6db4
 	github.com/pkg/errors v0.9.1
+	github.com/stolostron/multiclusterhub-operator v0.0.0-20220111202315-a556785ace10
```

Comment:
I think we should avoid importing github.com/stolostron/multiclusterhub-operator in the backplane. It seems to be adding some ugliness to our dependencies with client-go and mcho is already importing backplane-operator.